examples/read_pem_files/main.rs

```diff
@@ -0,0 +1,69 @@
+use config::{Config, Format, Map, Value, ValueKind, FileStoredFormat};
+use std::io::{Error, ErrorKind};
+
+/// The private and public key sources will be read into their associated variable:
+#[derive(serde::Deserialize, Clone, Debug)]
+pub struct Settings {
+    pub private_key: Option<String>,
+    pub public_key: Option<String>,
+}
+
+fn main() {
+    // Sourcing from two separate files for the `Settings` struct,
+    // these are both marked as optional to avoid failure if the file doesn't exist.
+    let file_public_key = config::File::new("public.pem", PemFile).required(false);
+    let file_private_key = config::File::new("private.pem", PemFile).required(false);
+
+    // Provide the sources and build the config object:
+    let settings = Config::builder()
+        .add_source(public_key)
```

Comment:
```suggestion
        .add_source(file_public_key)
```

Reply:
Yes, thank you these two sources are my fault as after the copy/pasting, I changed the variable names in the review suggestion and totally forgot to update them here 🤦‍♂️